lib/helpers/resource-node.js

```diff
@@ -9,7 +9,7 @@ module.exports = function resourceNode(name, options) {
     builders.callExpression(
       builders.memberExpression(
         builders.thisExpression(),
-        builders.identifier('resource'),
+        builders.identifier('route'),
```

Comment:
We already have a route-node.js which does this, we should probably just get rid of resource-node.js and all the logic related with resource.